src/lor_tools.jl

```diff
@@ -10,65 +10,65 @@ for_nodes(::Union{Ferrite.Lagrange{2,Ferrite.RefTetrahedron,2},Ferrite.Discontin
     (3,6,5),
     (5,4,2),
 )
-# Cubic Triangle
-for_nodes(::Union{Ferrite.Lagrange{2,Ferrite.RefTetrahedron,3},Ferrite.DiscontinuousLagrange{2,Ferrite.RefTetrahedron,3},Ferrite.Cell{2,10,3}}) = (
```

Comment:
Note that this also includes the refinement for interpolations. Maybe we just want to remove the Cell from the type union for now. Same for ones below.